app/models/admin/developer.rb

```diff
@@ -0,0 +1,8 @@
+class Admin::Developer < ApplicationRecord
+
+  attachment :profile_image
+
+  validates :name, :link, :function, :institution, :start_on_project, presence: true
+  validates_presence_of :leave_project, :if => lambda { |o| o.active == false } 
```

Comment:
This is a old syntax, see the [Rails Guides](http://guides.rubyonrails.org/active_record_validations.html#using-a-symbol-with-if-and-unless)